pkg/gitconfig/configs.go

```diff
@@ -145,6 +145,24 @@ func (cs *Configs) loadGlobalConfigs() string {
 		locs = append(locs, filepath.Join(appdir.UserHome(), cs.GlobalConfig))
 	}
 
+	// if we already have a global config we can just reload it instead of trying all locations
+	// but since gitconfig.New() already sets the global path to the globalConfigFile() one, we need to check
+	// its raw length to be sure it wasn't just the default empty config struct and was previously loaded
+	if cs.global != nil && cs.global.raw.Len() > 0 {
```

Comment:
Maybe we want to add something like `IsEmpty()` to the config struct? Would it make this easier to read?

Reply:
Done